licenses.html

```diff
@@ -2989,7 +3030,7 @@ <h4>Used by:</h4>
       same &quot;printed page&quot; as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2023 The OpenTelemetry Authors
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Is this related to the needed update to `cargo-about` that we can't apply due to the pinned version of rust?

Reply:
Ugh, maybe. Looking into it.

Reply:
@lennyburdette pointed out that this same string appears [on main](https://github.com/apollographql/router/blob/main/licenses.html) as well.

Reply:
We believe this was just the non-deterministic ordering in the file.